<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsSetting</class>
 <widget class="QWidget" name="QtWidgetsSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsSetting</string>
  </property>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>766</x>
     <y>34</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(255, 255, 255);
border:none;
border-image:url(:/Exit/resource/QtWidgetsSetting/Exit/exit0.png)
}
QPushButton:hover {
border-image:url(:/Exit/resource/QtWidgetsSetting/Exit/exit1.png)
}
QPushButton:pressed {
border-image:url(:/Exit/resource/QtWidgetsSetting/Exit/exit1.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BackgroundImageRect">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>524</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="QtWidgetsSetting.qrc">:/BackgroundImage/resource/QtWidgetsSetting/bg.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>449</y>
     <width>190</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(255, 255, 255);
color:rgb(220, 188, 96);
border:1px solid rgb(204, 204, 204);
border-radius:4px;
}
QPushButton:hover {
background-color:rgb(251, 248, 239);
}
QPushButton:pressed {
background-color:rgb(236, 233, 224);
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>449</y>
     <width>190</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(57, 59, 64);
color:rgb(244, 216, 168);
border:none;
border-radius:4px;
}
QPushButton:hover {
background-color:rgb(77, 79, 83);
}
QPushButton:pressed {
background-color:rgb(57, 59, 64);
}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UI_GeneralSetting">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>87</y>
     <width>218</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(246, 245, 243, 0);
color:rgb(117, 118, 121);
border:none;
}
QPushButton:hover {
background-color:rgb(244, 244, 244);
color:rgb(57, 59, 63);
}
QPushButton:pressed {
background-color:rgb(233, 233, 233);
color:rgb(57, 59, 63);
}</string>
   </property>
   <property name="text">
    <string>通用设置              </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UI_GameSetting">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>137</y>
     <width>218</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(246, 245, 243, 0);
color:rgb(117, 118, 121);
border:none;
}
QPushButton:hover {
background-color:rgb(244, 244, 244);
color:rgb(57, 59, 63);
}
QPushButton:pressed {
background-color:rgb(233, 233, 233);
color:rgb(57, 59, 63);
}</string>
   </property>
   <property name="text">
    <string>游戏设置              </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UI_ModuleSetting">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>187</y>
     <width>218</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(246, 245, 243, 0);
color:rgb(117, 118, 121);
border:none;
}
QPushButton:hover {
background-color:rgb(244, 244, 244);
color:rgb(57, 59, 63);
}
QPushButton:pressed {
background-color:rgb(233, 233, 233);
color:rgb(57, 59, 63);
}</string>
   </property>
   <property name="text">
    <string>模组设置              </string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleImageRect">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>9</y>
     <width>218</width>
     <height>78</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="QtWidgetsSetting.qrc">:/UI/resource/QtWidgetsSetting/UI/title.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabelRect">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UI_ModuleResource">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>237</y>
     <width>218</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(246, 245, 243, 0);
color:rgb(117, 118, 121);
border:none;
}
QPushButton:hover {
background-color:rgb(244, 244, 244);
color:rgb(57, 59, 63);
}
QPushButton:pressed {
background-color:rgb(233, 233, 233);
color:rgb(57, 59, 63);
}</string>
   </property>
   <property name="text">
    <string>模组资源              </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UI_VersionInformation">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>287</y>
     <width>218</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(246, 245, 243, 0);
color:rgb(117, 118, 121);
border:none;
}
QPushButton:hover {
background-color:rgb(244, 244, 244);
color:rgb(57, 59, 63);
}
QPushButton:pressed {
background-color:rgb(233, 233, 233);
color:rgb(57, 59, 63);
}</string>
   </property>
   <property name="text">
    <string>版本信息              </string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>87</y>
     <width>561</width>
     <height>339</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollArea{
background-color:rgb(255, 255, 255);
background:rgb(255, 255, 255);
border: none;
}

QScrollBar:vertical {
border: none;
background: rgb(255, 255, 255);
width: 6px;
margin: 0px 0 0px 0;
}
QScrollBar:horizontal {
border: none;
background: rgb(255, 255, 255);
height: 1px;
margin: 0px 0 0px 0;
}

QScrollBar::handle:vertical {
background: rgb(233, 236, 240);
border-radius: 0px;
min-height: 100px;
border: none;
}

QScrollBar::handle:horizontal {
background: rgb(244, 242, 238);
border-radius: 2px;
min-width: 520px;
border: none;
}

QScrollBar::handle:vertical:hover,QScrollBar::handle:vertical:pressed{ 
background:rgb(210, 213, 216);
}
QScrollBar::handle:horizontal:hover,QScrollBar::handle:horizontal:pressed{ 
background:rgb(210, 213, 216);
}

QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical{
background: none;
width: 0px;
height: 0px;
}
QScrollBar::add-page:horizontal,QScrollBar::sub-page:horizontal{
background: none;
width: 0px;
height: 0px;
}

QScrollBar::sub-line:vertical{
border: none;
height: 0px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal{
border: none;
width: 0px;
subcontrol-position: left;
subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal:hover,QScrollBar::sub-line:horizontal:pressed{
background: rgb(255, 255, 255);
width: 0px;
height: 0px;
}
QScrollBar::sub-line:horizontal:hover,QScrollBar::sub-line:horizontal:pressed{
background: rgb(255, 255, 255);
width: 0px;
height: 0px;
}

QScrollBar::add-line:vertical{
border: none;
height: 0px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::add-line:horizontal{
border: none;
width: 0px;
subcontrol-position: right;
subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover,QScrollBar::add-line:vertical:pressed{
background: rgb(255, 255, 255);
width: 0px;
height: 0px;
}
QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:pressed{
background: rgb(255, 255, 255);
width: 0px;
height: 0px;
}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-429</y>
      <width>561</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>561</width>
      <height>1000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color:rgb(255, 255, 255);
}</string>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>531</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
color:rgb(57, 59, 64);
spacing: 5px;
}
QCheckBox::indicator {
        width: 24px;
        height: 24px;

}
</string>
     </property>
     <property name="text">
      <string>开机自动启动</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>541</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(57, 59, 64);</string>
     </property>
     <property name="text">
      <string>启动设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>79</y>
       <width>541</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(57, 59, 64);</string>
     </property>
     <property name="text">
      <string>关闭设置</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>531</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>最小化到系统托盘</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>退出启动器</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>541</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(57, 59, 64);</string>
     </property>
     <property name="text">
      <string>本地文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>254</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 255, 255);
color:rgb(220, 188, 96);
border:1px solid rgb(204, 204, 204);
border-radius:4px;
}
QPushButton:hover {
background-color:rgb(251, 248, 239);
}
QPushButton:pressed {
background-color:rgb(236, 233, 224);
}</string>
     </property>
     <property name="text">
      <string>查找游戏启动器</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>521</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>521</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>541</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(57, 59, 64);</string>
     </property>
     <property name="text">
      <string>本地模组</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>254</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 255, 255);
color:rgb(220, 188, 96);
border:1px solid rgb(204, 204, 204);
border-radius:4px;
}
QPushButton:hover {
background-color:rgb(251, 248, 239);
}
QPushButton:pressed {
background-color:rgb(236, 233, 224);
}</string>
     </property>
     <property name="text">
      <string>选择模组文件夹</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>521</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>254</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 255, 255);
color:rgb(220, 188, 96);
border:1px solid rgb(204, 204, 204);
border-radius:4px;
}
QPushButton:hover {
background-color:rgb(251, 248, 239);
}
QPushButton:pressed {
background-color:rgb(236, 233, 224);
}</string>
     </property>
     <property name="text">
      <string>刷新所有模组</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>521</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>254</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(255, 255, 255);
color:rgb(220, 188, 96);
border:1px solid rgb(204, 204, 204);
border-radius:4px;
}
QPushButton:hover {
background-color:rgb(251, 248, 239);
}
QPushButton:pressed {
background-color:rgb(236, 233, 224);
}</string>
     </property>
     <property name="text">
      <string>查找游戏本体</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>BackgroundImageRect</zorder>
  <zorder>pushButton_Exit</zorder>
  <zorder>pushButton_Cancel</zorder>
  <zorder>pushButton_OK</zorder>
  <zorder>pushButton_UI_GeneralSetting</zorder>
  <zorder>pushButton_UI_GameSetting</zorder>
  <zorder>pushButton_UI_ModuleSetting</zorder>
  <zorder>TitleImageRect</zorder>
  <zorder>TitleLabelRect</zorder>
  <zorder>pushButton_UI_ModuleResource</zorder>
  <zorder>pushButton_UI_VersionInformation</zorder>
  <zorder>scrollArea</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsSetting.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Exit</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsSetting</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
